docs/tutorial.rst

```diff
@@ -1549,7 +1549,8 @@ By default, the following variables are available for use:
     may use `$XONSH_GITSTATUS_* <envvars.html>`_ to customize the styling.
   * ``localtime``: The current, local time as given by ``time.localtime()``.
     This is formatted with the time format string found in ``time_format``.
-  * ``time_format``: A time format string, defaulting to ``"%H:%M:%S"``.
+  * ``time_format``: A time format string, defaulting to ``"%H:%M:%S"``. See section
```

Comment:
The line `see section ....` is common to all the prompt-fields available in the prompt. It can be put in new line